cgi-bin/DW/Hooks/EmbedWhitelist.pm

```diff
@@ -105,6 +105,8 @@ my %host_path_match = (
 
     "codepen.io"            => [ qr!^/enxaneta/embed/!, 1 ]
 
+    "vid.me"                => [ qr!^/e/!, 1 ]
```

Comment:
Go ahead and end this line with a comma too, so that the next site doesn't have to add it.